server/testbed/Makefile

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2022 ANSSI
+# SPDX-License-Identifier: Apache-2.0
+#
+hostbus := $(shell lsusb | grep Bluetooth | sed -r 's/.*Bus ([0-9]+){1}.*/\1/')
+hostaddr := $(shell lsusb | grep Bluetooth | sed -r 's/.*Device ([0-9]+){1}.*/\1/')
+
+all: install scratch-build run
+quick: install cache-build run
+debug: install debug-build run
+
+scratch-build:
+	sudo mkosi -ff
+
+cache-build:
+	sudo mkosi --incremental -f
+
+debug-build:
+	sudo mkosi --kernel-command-line="systemd.log_level=debug systemd.log_target=console" --force
+
+install:
+	mkdir -p mkosi.extra/usr/lib/dracut/modules.d
+	cp -r ../dracut/90ultrablue mkosi.extra/usr/lib/dracut/modules.d/
+	mkdir -p mkosi.extra/usr/lib/systemd/system/
+	cp ../unit/ultrablue-server.service mkosi.extra/usr/lib/systemd/system/
+	mkdir -p mkosi.extra/etc
+	cp ressources/crypttab mkosi.extra/crypttab
+
+run:
+	mkdir -p /tmp/emulated_tpm/ultrablue
+	
+	swtpm socket \
+	          --tpmstate dir=/tmp/emulated_tpm/ultrablue \
+	          --ctrl type=unixio,path=/tmp/emulated_tpm/ultrablue/swtpm-sock \
+	          --log level=20 --tpm2 --daemon
+	
+	sudo mkosi qemu \
+	    -chardev socket,id=chrtpm,path=/tmp/emulated_tpm/ultrablue/swtpm-sock \
+	    -tpmdev emulator,id=tpm0,chardev=chrtpm \
+	    -device tpm-tis,tpmdev=tpm0 \
+	    -usb -device usb-host,hostbus=${hostbus},hostaddr=${hostaddr}
+
```

Comment:
Would be nice to add a `make clean` target. Could be in a separate commit.

Reply:
I don't want to add a clean target as it will needs to remove file with root privileges